### PR TITLE
STREAMS-1335 doc(): remove security section

### PR DESCRIPTION
**Pull request by dominiquemongelli**
https://deploy-preview-44--streams-open-docs.netlify.app

Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Remove security section. Keep only SDL section and a link to the security guide which required login

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

### CODE REVIEW

File: content/en/docs/security/_index.md [hunk 1]
```diff
@@ -3,18 +3,10 @@ title: Streams Security Guidance
 linkTitle: Security Guidance
 weight: 8
 date: 2020-09-18
-description: This guide describes the security features of Streams and provides instructions and best practices for strengthening its security.
+description: This section describes the security features of Streams and provides instructions and best practices for strengthening its security.
 ---
 
-This guide provides instructions and recommendations to help you strengthen the security of Streams. Security descriptions include:
+This section provides informations about:
```
Comment (allandenis38) [inline content/en/docs/security/_index.md:9]: This section provides information about

**Review by lbadenhop — APPROVED** [review]

**Comment by lbadenhop** [comment]
@dominiquemongelli @allandenis38 The following topic description is no longer true, "This section describes the security features of Streams and provides instructions and best practices for strengthening its security." The security guide now describes the security features and best practices. This section only describes how the product was developed in a secure way. I suggest changing the description.

**Review by allandenis38 — APPROVED** [review]

**Comment by dominiquemongelli** [comment]
@lbadenhop @allandenis38 I have updated the description and refactored a little bit the structure of the page.

**Review by allandenis38 — APPROVED** [review]

**Comment by lbadenhop** [comment]
Looks good.

**Review by lbadenhop — COMMENTED** [review]
Looks good

**Pull request merged**